packages/block-library/src/query-pagination-next/index.php

```diff
@@ -27,9 +27,15 @@ function render_block_core_query_pagination_next( $attributes, $content, $block
 	$wrapper_attributes = get_block_wrapper_attributes();
 	$show_label         = isset( $block->context['showLabel'] ) ? (bool) $block->context['showLabel'] : true;
 	$default_label      = __( 'Next Page' );
-	$label_text         = isset( $attributes['label'] ) && ! empty( $attributes['label'] ) ? esc_html( $attributes['label'] ) : $default_label;
-	$label              = $show_label ? $label_text : '';
-	$pagination_arrow   = get_query_pagination_arrow( $block, true );
+	// Allowed HTML tags for the label text.
+	$allowed_tags = array(
+		'span' => array(),
```

Comment:
@ajitbohra I'm just wondering if there is value in including `i` tags to the list of allowed tags to allow for Font Awesome icons?